Projects/TEAM-3/Mykhailenko/QA2022_04_402/ex402/Tests/RegisterUser_Test.cs

```diff
@@ -0,0 +1,33 @@
+﻿/*
+ Testing local OpenCart Bitnami
+*/
+
+using NUnit.Framework;
+using ex402.Pages;
+using ex402.Utils;
+
+namespace ex402.Tests
+{
+    [TestFixture]
+    internal class RegisterUser_Test : Values
```

Comment:
RegisterUser_Test: Values - it is an interesting approach but not the best

Reply:
In final framework Values is static class
Values class stored started value in one pleace